src/main/groovy/pl/allegro/tech/build/axion/release/infrastructure/git/GitRepository.groovy

```diff
@@ -176,13 +185,24 @@ class GitRepository implements ScmRepository {
     TagsOnCommit latestTags(Pattern pattern, String sinceCommit) {
         return latestTagsInternal(pattern, sinceCommit, false)
     }
+		
+		private class TagHolder {
```

Comment:
also we might not need this at all if this repository simply returns list (ordered map) of all commits & tags associated with them